app/services/decisions/client_decision_tree.rb

```diff
@@ -30,6 +30,14 @@ def after_has_partner
       end
     end
 
+    def after_client_details
+      if IojPassporter.new(form_object).applicant_under18_passport?
```

Comment:
I think the `IojPassporter` class is initialised with a crime application instead the form object. In this case as both objects respond to `#applicant` it works, but might be best to change it to avoid future problems 😄

Reply:
👍 Thank you!